questionnaires/superset/datasets.py

```diff
@@ -16,7 +16,7 @@ def post_body(self):
         }
 
     def put_body(self, columns, metrics):
-        sql = f"SELECT * " \
+        sql = f"SELECT id, form_data, submit_time, page_id, user_id " \
```

Comment:
Why not use `ALLOWED_COLUMNS` here? It would eliminate the need to change the column list in two separate places.

Reply:
Fixed it.